src/sgame/sg_bot_skilltree.cpp

```diff
@@ -24,6 +24,39 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
 #include "sg_bot_util.h"
+#include "../shared/parse.h"
+
+static std::set<std::string> bg_allowedSkillset;
+
+static std::set<std::string> BG_ParseAllowedSkillsetList( const std::string &behaviorsCsv )
```

Comment:
This is sgame, so the convention is to use a `G_` prefix (nitpick)

```suggestion
static std::set<std::string> G_ParseAllowedSkillsetList( const std::string &behaviorsCsv )
```